wiring/inc/spark_wiring_wifi_credentials.h

```diff
@@ -59,17 +59,17 @@ enum SecurityType {
 class WiFiCredentials {
 public:
     WiFiCredentials(SecurityType security = UNSEC) {
-        setSecurity((WLanSecurityType)security);
+        setSecurity((SecurityType)security);
```

Comment:
Why do we need a cast here and the other places? It's the same type.

Reply:
That's right - the places in this file can be gotten rid of. 
The other couple places fails compiling because 
1. `toSecurity` returns a `WLANSecurityType`. (https://github.com/particle-iot/device-os/blob/27a0177608ef33014bf9a02489714fb8f5adcc57/hal/src/photon/softap.cpp#L638)
2. This as well `security_` is of `WLANSecurityType`

Reply:
That one is correct :+1: